src/main/java/org/testcontainers/containers/ExecResultFactory.java

```diff
@@ -5,14 +5,14 @@
  */
 public class ExecResultFactory {
```

Comment:
Explain that this is need to create an ExecResult because it's package-private.
Update class comment (it refers to mock tests).